css/main.css

```diff
@@ -458,6 +461,100 @@ body {
   margin: 18px 0;
 }
 
+.modal {
+  background: #fff;
+  padding: 1.5rem;
+  width: 90%;
+  opacity: 0;
+  visibility: hidden;
+  transition: transform 1s, opacity 1s, visibility 0s;
+  z-index: 300;
+}
```

Comment:
- [ ] You did a good job on the project functionalities but for this project, you are supposed to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design as much as possible. Please make sure the image in the model occupies almost 95% of the model width and it matches the design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/167875525-1ba6b108-5f63-4b8d-9ae6-e671857f00e4.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/167875449-38ad54b9-e209-4dca-a680-65cfc19a3363.png)

Reply:
ok, thank for helpful advice I will follow